### DISCUSSION TITLE
Avoid (Ontological) Weasel Words in Definitions (unnecessary phrases or clauses that merely suggest a boundary)

### DISCUSSION THREAD

**Discussion by jonathanvajda** [CommonCoreOntology/CommonCoreOntologies#366] [Ontology Terms and Class Schema]
I had conversation with the CUBRC team offline, but I was told to make this request public.

Definitions should avoid examples and intentionally vague language. This isn't an inviolable rule, but one we can avoid in a mid-level ontology. There are cagey words that give wiggle room, and we can avoid them. Terms and phrases like:
- example
- such as
- in other words
- for illustration
- for instance
- typical*
- usual*
- mostly
- often

Examples are supposed to be annotated with cco:example_of_usage
Elucidations are supposed to be annotated with cco:elucidation

For the mid-level ontologies, here's what my QC SPARQL query pulled:
- http://www.ontologyrepository.com/CommonCoreOntologies/affects
- http://www.ontologyrepository.com/CommonCoreOntologies/interval_disjoint
- http://www.ontologyrepository.com/CommonCoreOntologies/is_affected_by
- http://www.ontologyrepository.com/CommonCoreOntologies/ActOfAssignment
- http://www.ontologyrepository.com/CommonCoreOntologies/ActOfMassMediaCommunication
- http://www.ontologyrepository.com/CommonCoreOntologies/ActOfOathTaking
- http://www.ontologyrepository.com/CommonCoreOntologies/ActOfReligiousWorship
- http://www.ontologyrepository.com/CommonCoreOntologies/ActOfSojourn
- http://www.ontologyrepository.com/CommonCoreOntologies/ArmoredPersonnelCarrier
- http://www.ontologyrepository.com/CommonCoreOntologies/BeverageAntenna
- http://www.ontologyrepository.com/CommonCoreOntologies/BiologicalWeapon
- http://www.ontologyrepository.com/CommonCoreOntologies/Blue
- http://www.ontologyrepository.com/CommonCoreOntologies/Bullet
- http://www.ontologyrepository.com/CommonCoreOntologies/Cannon
- http://www.ontologyrepository.com/CommonCoreOntologies/City
- http://www.ontologyrepository.com/CommonCoreOntologies/ColorHue
- http://www.ontologyrepository.com/CommonCoreOntologies/ColorSaturation
- http://www.ontologyrepository.com/CommonCoreOntologies/CommunicationSystem
- http://www.ontologyrepository.com/CommonCoreOntologies/CriminalAct
- http://www.ontologyrepository.com/CommonCoreOntologies/CryogenicStorageDewar
- http://www.ontologyrepository.com/CommonCoreOntologies/Cyan
- http://www.ontologyrepository.com/CommonCoreOntologies/DesignativeName
- http://www.ontologyrepository.com/CommonCoreOntologies/EthnicGroup
- http://www.ontologyrepository.com/CommonCoreOntologies/Ethnicity
- http://www.ontologyrepository.com/CommonCoreOntologies/Evening
- http://www.ontologyrepository.com/CommonCoreOntologies/FacialHair
- http://www.ontologyrepository.com/CommonCoreOntologies/FlightTransponder
- http://www.ontologyrepository.com/CommonCoreOntologies/FuelCell
- http://www.ontologyrepository.com/CommonCoreOntologies/Green
- http://www.ontologyrepository.com/CommonCoreOntologies/Hazel
- http://www.ontologyrepository.com/CommonCoreOntologies/HeavyMachineGun
- http://www.ontologyrepository.com/CommonCoreOntologies/HelicalAntenna
- http://www.ontologyrepository.com/CommonCoreOntologies/HornAntenna
- http://www.ontologyrepository.com/CommonCoreOntologies/InstantMessaging
- http://www.ontologyrepository.com/CommonCoreOntologies/LegalArtifactFunction
- http://www.ontologyrepository.com/CommonCoreOntologies/LightMachineGun
- http://www.ontologyrepository.com/CommonCoreOntologies/Loudness
- http://www.ontologyrepository.com/CommonCoreOntologies/Mailing
- http://www.ontologyrepository.com/CommonCoreOntologies/MediumMachineGun
- http://www.ontologyrepository.com/CommonCoreOntologies/ModePointEstimateInformationContentEntity
- http://www.ontologyrepository.com/CommonCoreOntologies/Motorcycle
- http://www.ontologyrepository.com/CommonCoreOntologies/NavigationSystem
- http://www.ontologyrepository.com/CommonCoreOntologies/Night
- http://www.ontologyrepository.com/CommonCoreOntologies/NonNameIdentifier
- http://www.ontologyrepository.com/CommonCoreOntologies/Orange
- http://www.ontologyrepository.com/CommonCoreOntologies/ParamilitaryForce
- http://www.ontologyrepository.com/CommonCoreOntologies/Park
- http://www.ontologyrepository.com/CommonCoreOntologies/PatchAntenna
- http://www.ontologyrepository.com/CommonCoreOntologies/PayloadCapacity
- http://www.ontologyrepository.com/CommonCoreOntologies/Pitch
- http://www.ontologyrepository.com/CommonCoreOntologies/PoliticalOrientation
- http://www.ontologyrepository.com/CommonCoreOntologies/PrivateNetworkTelephoneCall
- http://www.ontologyrepository.com/CommonCoreOntologies/PropellingNozzle
- http://www.ontologyrepository.com/CommonCoreOntologies/PublicSafetyArtifactFunction
- http://www.ontologyrepository.com/CommonCoreOntologies/RadioRepeater
- http://www.ontologyrepository.com/CommonCoreOntologies/RandomWireAntenna
- http://www.ontologyrepository.com/CommonCoreOntologies/Red
- http://www.ontologyrepository.com/CommonCoreOntologies/RhombicAntenna
- http://www.ontologyrepository.com/CommonCoreOntologies/SatelliteArtifact
- http://www.ontologyrepository.com/CommonCoreOntologies/Shotgun
- http://www.ontologyrepository.com/CommonCoreOntologies/SubmachineGun
- http://www.ontologyrepository.com/CommonCoreOntologies/Tattoo
- http://www.ontologyrepository.com/CommonCoreOntologies/Texture
- http://www.ontologyrepository.com/CommonCoreOntologies/Timbre
- http://www.ontologyrepository.com/CommonCoreOntologies/Town
- http://www.ontologyrepository.com/CommonCoreOntologies/Trail
- http://www.ontologyrepository.com/CommonCoreOntologies/TrainingCamp
- http://www.ontologyrepository.com/CommonCoreOntologies/Village
- http://www.ontologyrepository.com/CommonCoreOntologies/Violet
- http://www.ontologyrepository.com/CommonCoreOntologies/Wetness
- http://www.ontologyrepository.com/CommonCoreOntologies/WiredCommunicationArtifactFunction
- http://www.ontologyrepository.com/CommonCoreOntologies/Yellow

**Comment by swartik** [comment]
@jonathanvajda I hope future versions of CCO incorporate your points. They should be in the Best Practices of Ontology Development document.

**Comment by neilotte** [comment]
@jonathanvajda I've started a thread in the discussion board section regarding the creation of a style guide. Can we move this issue under that heading and close here? Thanks! 

https://github.com/CommonCoreOntology/CommonCoreOntologies/discussions/317

**Comment by alanruttenberg** [comment]
Sometimes intentionally vague language is necessary, as some boundaries are essentially vague.
